spec/index.html

```diff
@@ -242,7 +243,7 @@ <h2>A Canonical form of N-Quads</h2>
 
   <p>This section defines a canonical form of N-Quads which has
     a completely specified layout.
-    The grammar for the language is the same.</p>
+    The grammar for the language remains unchanged.</p>
```

Comment:
As in N-Triples, I think this is the better tweak:
```suggestion
    The grammar for the language is unchanged.</p>
```

Reply:
In RDF Canonicalization and in N-Triples we use "remains unchanged". We should try to use the same wording where possible.

Reply:
> In RDF Canonicalization and in N-Triples we use "remains unchanged".

I suggested `is unchanged` on N-Triples before doing so here.  I hadn't tripped over the RDF Canonicalization instance yet.

> We should try to use the same wording where possible.

Yes. And as I noted in both N-Triples and N-Quads, I think `is unchanged` is the better option.

Reply:
Yes, indeed you did, and it was accepted. I'll do so here.